### PR TITLE
DDS Subscriber get/set qos

### PR DESCRIPTION
**Pull request by IkerLuengo**
- Correct get_qos on PIM
- Unit test for PIM Subscriber get/set_qos
- Link PSM qos getter/setter to PIM implementation
- Unit test for PSM Subscriber qos getter/setter

### CODE REVIEW

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/1742/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/1499/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/1734/)

File: test/unittest/dds/subscriber/SubscriberTests.cpp [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+{
+    DomainParticipant* participant = DomainParticipantFactory::get_instance()->create_participant(0);
+    Subscriber* subscriber = participant->create_subscriber(SUBSCRIBER_QOS_DEFAULT);
+
+    SubscriberQos qos;
+    ASSERT_EQ(subscriber->get_qos(qos), ReturnCode_t::RETCODE_OK);
+
+    ASSERT_EQ(qos, SUBSCRIBER_QOS_DEFAULT);
+
+    qos.entity_factory.autoenable_created_entities = false;
```
Comment (laura-eprosima) [inline test/unittest/dds/subscriber/SubscriberTests.cpp:37]:
```suggestion
    qos.entity_factory().autoenable_created_entities = false;
```

Comment (laura-eprosima) [reply test/unittest/dds/subscriber/SubscriberTests.cpp:37]: The same for the ones below

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/1856/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/1598/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/1850/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/1865/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/1606/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/1858/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/1872/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/1613/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/1865/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/1930/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/1668/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/1924/)

**Pull request merged**